### ISSUE TITLE
The ff ligature has uneven cross strokes

### ISSUE THREAD

**Issue by atheistray** [huertatipografica/Alegreya-Sans#7]
I'm using FB reader on my phone, which is running Android 5.1.
You can see in the picture that the f on the right appears higher than its counterpart.
This problem occurs at some font sizes. For some sizes, they look fine, but for others, they look uneven.
![screenshot_2017-09-30-09-11-44-69](https://user-images.githubusercontent.com/22341522/31041842-aa5c7f5c-a561-11e7-8de8-26603bb9c05d.png)
![screenshot_2017-09-30-11-00-20-88](https://user-images.githubusercontent.com/22341522/31041847-b27deea0-a561-11e7-8e52-53e084384e7a.png)

**Comment by juandelperal** [comment]
Hi!
Thanks for writing us.
It seems to be a problem of the app or maybe they are using an old version
of the font.
The actual is working good:

[image: Imágenes integradas 1]

Best regards

2017-09-30 0:01 GMT-03:00 atheistray <notifications@github.com>:

**Comment by atheistray** [comment]
It's not the app nor the font version. I am using the latest otf fonts downloaded from here and after I tested it in browser environment I got the same result. The ft ligature also has the same problem.
These screenshots were taken in opera browser and chrome browser.
![screenshot_2017-10-03-11-02-19-24](https://user-images.githubusercontent.com/22341522/31108842-57d77b4e-a7c1-11e7-9f12-dd18dfa9f0ae.png)
![screenshot_2017-10-03-11-03-56-36](https://user-images.githubusercontent.com/22341522/31108855-70ad9298-a7c1-11e7-8141-892ed8e0aef5.png)
![screenshot_2017-10-03-11-25-11-71](https://user-images.githubusercontent.com/22341522/31108863-7ebbacee-a7c1-11e7-8796-0513ab0503c1.png)
![screenshot_2017-10-03-11-26-22-34](https://user-images.githubusercontent.com/22341522/31108870-8da977fe-a7c1-11e7-98ad-17e8685f9d92.png)

**Comment by juandelperal** [comment]
Hi @atheistray 
Now I released a new version, I detected a problem in the t, but the rest of the glyphs were OK. I am updating all the fonts today. If you detect that the problem persist please let me know. Thanks for the feedback!

**Comment by atheistray** [comment]
Sorry for late reply. I did some more testing using the latest otf files you updated five days ago. The problem is still there. But I found out that if I turn off the hinting on FB reader, it fixes the problem:
With hinting on:
![screenshot_2017-11-19-21-49-47-28](https://user-images.githubusercontent.com/22341522/32991515-b23b3038-cd02-11e7-94eb-b6ceeda7b605.png)
With hinting off
![screenshot_2017-11-19-21-49-59-19](https://user-images.githubusercontent.com/22341522/32991520-d43a4d86-cd02-11e7-8fd0-74894b51e547.png)
However, for web browser, the problem persists.
![screenshot_2017-11-19-22-12-54-19](https://user-images.githubusercontent.com/22341522/32991526-f12bbeca-cd02-11e7-8f05-647f30960a6e.png)
![screenshot_2017-11-19-22-14-54-26](https://user-images.githubusercontent.com/22341522/32991528-f67cd0d0-cd02-11e7-86c1-2b14060c59c6.png)
Now, I guess this problem is related to how Android handles font hinting maybe?
